src/components/HomePage/Description.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Box, Container, Typography } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import Text from "./TextFields";
+import { Link } from "react-router-dom";
+
+interface textProps {
+  link: string;
+  title: string;
+  text1: string;
+  text2: string;
```

Comment:
please try to avoid this naming pattern with numbers, change `text1`, `text2` to something more self describing